INSTALL.md

```diff
@@ -211,43 +212,66 @@ If you have "error: undefined reference to 'stdout'" or so, make sure that in BU
+
+   - Open QtCreator and navigate to `settings -> devices -> Android`, here:
+      - JDK location: Add Path to Java version, e.g. `/opt/homebrew/Cellar/openjdk@17/17.0.11/libexec/openjdk.jdk/Contents/Home`
+      - Android SDK location: set path to some empty writeable directory, e.g. `~/android`
+      - Hit `Set up SDK` and install the current SDK version (find the correct version in `.github/workflows/android.yml`)
+      - Let QtCreator install NDK
+      - Let QtCreator install openssl
+   - QtCreator should now say `Android settings are OK.`
+
+4. Setup Android SDK & NDK [Manual, via sdkmanager]
+   - Proceed with this step only if the previous automatic step did not work for you or you do not want to use QtCreator
+
+   - Get Android `sdkmanager` by following these steps https://developer.android.com/tools/sdkmanager
+   - Now perform `./cmdline-tools/bin/sdkmanager --sdk_root=./ "build-tools;33.0.1" "ndk;25.2.9519653" "platforms;android-33" platform-tools tools` to install all needed Android tools, make sure to double-check if the version numbers are correct
+
+5. Get MM mobile SDK - it contains pre-built dependencies of used libraries (QGIS, etc..)
+
+   - Check what SDK version is currently in use - look for `INPUT_SDK_VERSION` in `.github/workflows/android.yml`
+   - Download TWO SDKs for android (arm and arm64) - go to https://github.com/merginmaps/mobile-sdk/releases and download the built SDK.
+   - Unpack the downloaded .tar.gz to `~/mobile-sdk/arm-android` and `~/mobile-sdk/arm64-android`
+   - WARNING!! It is super important to have both SDKs in same subfolder (e.g. `~/mobile-sdk`) and have folder name `arm64-android` and `arm-android`
+
+6. Build (update CMake command with the correct Qt and SDK versions)
 
-4. Build Input (update CMake command with the correct Qt and Input SDK versions)
 ```
-  export ANDROID_SDK_ROOT=/opt/Android/android-sdk;
-  export ANDROID_NDK_ROOT=/opt/Android/android-sdk/ndk/25.1.8937393;
-  export QT_BASE=/opt/Qt/6.5.2;
+  # Needed Android variables
+  export ANDROID_SDK_ROOT=~/android;
+  export ANDROID_NDK_ROOT=~/android/ndk/25.1.8937393;
   export ANDROID_NDK_PLATFORM=android-24;
-  export INPUT_SDK_ANDROID_BASE=~/Projects/quick/input-sdk/build/android/stage;
-  export PATH=/usr/local/Cellar/openjdk\@11/11.0.16.1_1/bin/:$PATH;
+
+  # INPUT_SDK_ANDROID_BASE is a path where you stored the two SDKs from the mobile-sdk repo
+  export INPUT_SDK_ANDROID_BASE=~/mobile-sdk;
+
+  # (optional, not needed often) add Java to PATH if you need to use other Java version than your default one
+  export PATH=/opt/homebrew/Cellar/openjdk@17/17.0.11/libexec/openjdk.jdk/Contents/Home/bin:$PATH;
+
   cmake \
-    -DCMAKE_BUILD_TYPE=Release \
-    -DQT_ANDROID_ABIS="arm64-v8a;armeabi-v7a" \
-    -DQT_HOST_PATH=$QT_BASE/macos \
-    -DCMAKE_TOOLCHAIN_FILE=$QT_BASE/android_arm64_v8a/lib/cmake/Qt6/qt.toolchain.cmake \
+    -DCMAKE_BUILD_TYPE=Debug \
```

Comment:
Does Debug work ? I always use RelWithDbInfo for mobile apps,  i think on some devices the debug build is not starting.

Reply:
I use debug all the time 🤔, didn't know it causes troubles on some machines

Reply:
I use release for iOS

Reply:
maybe it is for iOS and I am confused. If you use it is it fine.

Reply:
... to do release build on Android you need all these signing certificates, don't you?